tests/instruments/agilent/test_agilent33250A_with_device.py

```
@@ -0,0 +1,163 @@
+    return Agilent33250A(connected_device_address)
+
+
+@pytest.fixture(autouse=True)
+def force_output_off(generator):
+    """Ensure output is off before and after each test."""
+    generator.output_enabled = False
+    yield
+    try:
+        generator.output_enabled = False
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)